Makefile

```diff
@@ -119,4 +119,4 @@ clean:
 	@cd $(CURDIR) && cargo clean
 
 build_image:
-	docker build -t merqury/etsi_014_ref_impl -f Dockerfile .
+	docker build -t merqury/etsi_014_ref_impl:1.1.0 -f Dockerfile .
```

Comment:
Question: Does it make sense to have a docker compose service and corresponding make target which start the etsi-ref-impl as well?

Reply:
On second thoughts, I would advice against this given that we already have [this](https://github.com/cybermerqury/etsi-gs-qkd-014-referenceimplementation/blob/main/examples/run_server.sh) bash script to execute the application natively on the host machine. Adding dedicated containers to run the application might be a bit confusing for the users.

Reply:
I am on the fence about make targets that just call `docker compose up -d` personally as well. I can see how it could be nice to just interact with everything using `make`, but all docker related actions can also be handling through `docker` so arguably it is an abuse of `make`

Reply:
Resolved as per our discussion.